### DISCUSSION TITLE
covpass as RFID tag?

### DISCUSSION THREAD

**Discussion by dk5ras** [Digitaler-Impfnachweis/covpass-android#165] [Ideas]
Hi all, 

just a maybe stupid question...are there any plans implementing a format to store the covpass into an RFID tag? Or does this already exist? At least Android ahould allow even the covpass checker app to access the NFC module to read it. I came over the ID while printing the QR code onto a plastic card with MIFARE chip inside :)

Ralph.

Edit: Ich bin Deutscher, ggf. also auch gerne in Deutsch; hab' nur aus Gewohnheit in Englisch geschrieben, weil alle anderen Projekte, mit denen ich zu tun habe, in Englisch abgewickelt werden :)

**Comment by vaubaehn** [comment]
Hi @dk5ras , I'm not a developer, but I may throw some assumptions into the pot.

Over at the Corona-Warn-App repository, there are currently some evaluations on the way to enable check-ins via NFC stored on RFID tags for Android. I was a bit surprised that for iOS that was already working out-of-the-box. For Android it seems to be as easy as to add an NFC intent to the manifest.

So I would strongly suppose, what you're asking for is theoretically possible, given the RFID tag has a capacity of at least 800 bytes (to be on the safe side, as the average DCC has a size around 750 bytes).
If you're able to build CovPass or CovPassCheck yourself, with minimal effort that might be possible.

Keep us updated what was your outcome.
